src/App/index.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
+import Button from '../components/Button/index';
 
 
 function App(){
+    
     return(
-        <div></div>
+        <div>
+            <Button value={'click'} disabled={false}/>
```

Comment:
Please instead of providing the text content of the button in the value prop provide it as the children of the `Button` component.
Also the `disabled` prop of the button is supposed to be optional so no need to provide `disabled={false}` here.
```suggestion
            <Button>Click</Button>
```